OpenRA.Mods.Common/Traits/ExternalCaptures.cs

```diff
@@ -128,7 +128,7 @@ public override bool CanTargetActor(Actor self, Actor target, TargetModifiers mo
 		{
 			var c = target.TraitOrDefault<ExternalCapturable>();
 
-			var canTargetActor = c != null && !c.CaptureInProgress && c.Info.CanBeTargetedBy(self, target.Owner);
+			var canTargetActor = c != null && !c.CaptureInProgress && c.CanBeTargetedBy(self, target.Owner);
```

Comment:
`CanTargetFrozenActor` is still checking the `ExternalCapturableInfo`, and will need to be changed to match `CaptureOrderTargeter`.

Reply:
Now I think that it is correct not to change CanTargetFronzenActor, as the fog obscures the capturableness. Captures.cs is reverted.

Reply:
It needs to account for the state at the time the actor was frozen, otherwise you leak info through the fog.  I have a set of prs lined up to solve this, but so far #13995 has not had any review attention.

Reply:
On further thought, I don't think we should hold up this PR over the future frozen actor fixes.

So long as this PR doesn't change the behaviour for non-disabled traits then this can still be useful for regular units that don't get frozen under fog.